pom.xml

```diff
@@ -143,8 +143,11 @@
     <groovy-maven-plugin.version>2.0</groovy-maven-plugin.version>
     <surefire-plugin.version>2.20</surefire-plugin.version>
     <failsafe-plugin.version>2.20</failsafe-plugin.version>
+    <maven-compiler-plugin.version>3.6.1</maven-compiler-plugin.version>
```

Comment:
would prefer consistent naming. We seem to have both forms for `org.apache.maven.plugins:maven-x-plugin`:

* ${maven-x-plugin}
* ${x-plugin}